jlm/llvm/ir/operators/StoreNonVolatile.cpp

```diff
@@ -5,36 +5,37 @@
 
 #include <jlm/llvm/ir/operators/alloca.hpp>
 #include <jlm/llvm/ir/operators/operators.hpp>
-#include <jlm/llvm/ir/operators/store.hpp>
+#include <jlm/llvm/ir/operators/StoreNonVolatile.hpp>
 
 namespace jlm::llvm
 {
 
-StoreOperation::~StoreOperation() noexcept = default;
+StoreNonVolatileOperation::~StoreNonVolatileOperation() noexcept = default;
 
 bool
-StoreOperation::operator==(const operation & other) const noexcept
+StoreNonVolatileOperation::operator==(const operation & other) const noexcept
 {
-  auto operation = dynamic_cast<const StoreOperation *>(&other);
+  auto operation = dynamic_cast<const StoreNonVolatileOperation *>(&other);
   return operation && operation->narguments() == narguments()
       && operation->GetStoredType() == GetStoredType()
       && operation->GetAlignment() == GetAlignment();
 }
 
 std::string
-StoreOperation::debug_string() const
+StoreNonVolatileOperation::debug_string() const
 {
   return "STORE";
```

Comment:
Change this to "Store" like you did with "Load"?

Reply:
Fixed. Good catch!